### ISSUE TITLE
2.4.5 Wallet discriminates the payment size only allowing nano payments

### ISSUE THREAD

**Issue by raul7angels** [iotaledger/wallet#444]
Using latest Light wallet on Mac. I can make tiny transfers 1 - 1000 IOTA but once I go above these numbers I get "ERROR: PRIVATE KEY REUSE DETECTED!". 

If the problem was REALLY with the key, it should NOT discriminate between 1 IOTA vs 1 GIOTA, but it does even if I repeat the experiment 100 times.

What's up with your code guys?

**Comment by raul7angels** [comment]
One thing maybe worth mentionin is that I was trying out the MINEIOTADOTCOM, and got several > 20000 IOTA payments from them. One payment however has 8 duplicates, 7 of which are pending and 1 is confirmed. The problem with the PRIVATE KEY REUSE ERROR started even before that so it's probably unrelated.

**Comment by chrisdukakis** [comment]
It seems that you have received tokens to an address you have previously spent from. Wallet won't allow to spend from this address to prevent key reuse. I understand this is undesirable to you, but it is there to prevent users from accidentally reusing their keys.
Maybe you could try another wallet.

**Comment by raul7angels** [comment]
Thanks Chris

Perhaps it could be a more of a Question? "Are you sure you want to reuse the key? This will compromise your seed." For example for cases like mine when I wanted to close down a seed and get my funds unstuck? :)

--- Just for everyone else this is what went wrong with my wallet and how to fix it.

Hey guys

All the FAQ's say that "don't send from same address twice but you can receive to same address unlimited times" :)

So basically what happened was that i ordered my IOTA balance from Bitfinex to the wallet receive address. This process took 12 hours, so meanwhile I decided to make a new more secure seed and transferred to existing balance to the new wallet from the same address as I thought I can use this address for spending since it had never been used for spending before. **What I didn't realise was that this locks up this address that the Bitfinex funds are coming to.** Was able to do the complete transfer with Android Wallet that ignores the double spending from same address and close down the whole wallet that was now compromised. 

Lesson learned and the big benefit is that I now understand the Tangle much better than I did before.

Cheers and thanks for the help! Hopefully my stuggle will help someone else in the future.

Raul​​​​​​

**Comment by larrysmith1000** [comment]
Good Call Raul.  There should be a way to spend from a previously used address, because otherwise the funds are locked.

Perhaps a notification to the user explaining the risks, and then only all or none send transactions are aloud to the address gets retired for good.

**Comment by raul7angels** [comment]
Yes, great idea, only complete balance transfer is allowed since the whole seed will be comprimised... Great call Larry ;)
